plugins/org.eclipse.papyrus.uml.diagram.sequence.runtime/src/org/eclipse/papyrus/uml/diagram/sequence/runtime/internal/edit/parts/LifelineHeaderEditPart.java

```diff
@@ -69,6 +71,7 @@ protected void createDefaultEditPolicies() {
 
 		ResizableEditPolicy resizePolicy = new LifelineHeaderResizeEditPolicy();
 		installEditPolicy(EditPolicy.PRIMARY_DRAG_ROLE, resizePolicy);
+		installEditPolicy(EditPolicyRoles.SEMANTIC_ROLE, new LogicalModelElementDeletionSemanticEditPolicy());
```

Comment:
The "semantic" role name is more general than deletion, so probably the edit-policy should be named `LogicalModelElementSemanticEditPolicy`

Reply:
done